syntax = "proto3";

service Showtime {
    rpc GetShowMoviesByDate(ShowtimeDate) returns (Schedule) {}
    rpc GetListShowtimes(ShowtimeEmpty) returns (AllSchedule) {}
}

message ShowtimeDate {
    string date = 1;
}

message Schedule {
    string date = 1;
    repeated string movies = 2;
}

message AllSchedule {
    repeated Schedule schedules = 1;
}

message ShowtimeEmpty {
}
